examples/jsm/nodes/utils/OscNode.js

```diff
@@ -22,7 +22,7 @@ class OscNode extends Node {
 	construct() {
 
 		const method = this.method;
-		const timeNode = this.timeNode;
+		const timeNode = nodeObject( this.timeNode );
```

Comment:
Maybe we should assume that an user always passes a node object?

Reply:
I think better added it in `construct()` as default in case of user added a class based node like:

```js
oscSine( new CustomNode() );
```

Reply:
Maybe we can instruct the user to always pass node objects? E.g. like `oscSine( nodeObject( new CustomNode() ) )`?

Reply:
> Maybe we can instruct the user to always pass node objects? E.g. like oscSine( nodeObject( new CustomNode() ) )?

I think the best way would still be to standardize possible `non-nodeObjects` in `construct()` to facilitate implementation.

Anyway no object would be converted twice once we have a WeekMap.